py-polars/polars/exceptions.py

```diff
@@ -14,27 +14,35 @@
     # when there is no binary yet
 
     class ArrowError(Exception):  # type: ignore
+        """Exception raised the underlying Arrow library encounters an error"""
         pass
 
     class ComputeError(Exception):  # type: ignore
+        """Exception raised when evaluation of an expression fails"""
```

Comment:
Maybe make it a bit more broad? We throw this error on a lot of place, not only in expressions.

`Exception raised when we couldn't finish the computation`